src/plugins/intel_cpu/src/emitters/jit_load_store_emitters.cpp

```diff
@@ -15,6 +15,19 @@ using namespace dnnl::impl::cpu::x64;
 using namespace Xbyak;
 using namespace Xbyak::util;
 
+#define STORE_KEEP_SOURCE(is_updated, instruction, data_reg, data_reg_new, ...) \
```

Comment:
I would say this marco makes the code mich less readable. Can we at least add some comments that descibe its behavior. Also I wouls suggest to get rid of `is_updated` argument given it is always equal to `data_reg_updated`

Reply:
What about `is_updated` argument?

Reply:
I replaced macro parameter name from `is_updated `to `data_reg_updated ` along previous commit. I think I got your point now that remove this parameter at all from its signature. Done in [dd027ac](https://github.com/openvinotoolkit/openvino/pull/16313/commits/dd027ac791fdb5f975f798ce6f14e29d1cf3762b), take a look again please, thank you!